cluster-scope/base/core/namespaces/nfs-server-and-provisioner/namespace.yaml

```
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/display-name: "NFS Server and Provisioner"
+    openshift.io/requester: operate-first
+    op1st/project-owner: operate-first
+    op1st/onboarding-issue: "https://github.com/operate-first/support/issues/1119"
+    op1st/docs: "https://github.com/operate-first/apps/nfs-server-and-provisioner"
+  name: dex
```

This name should not be dex
```suggestion
  name: nfs-provisioner
```

---

good catch